lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb

```diff
@@ -6,60 +6,15 @@ module Fastlane
 
   module Helper
     module FirebaseAppDistributionHelper
-      def testers_flag(params)
-        file_flag_if_supplied("--testers-file", "testers", params)
-      end
-
-      def groups_flag(params)
-        file_flag_if_supplied("--groups-file", "groups", params)
-      end
-
-      def release_notes_flag(params)
-        file_flag_if_supplied("--release-notes-file", "release_notes", params)
-      end
-
-      def file_flag_if_supplied(flag, param_name, params)
-        file = params["#{param_name}_file".to_sym]
-        file ||= file_for_contents(param_name.to_sym, params)
-
-        if file
-          return "#{flag} #{file}"
+      # Should we throw an exception if path is empty or not passed in? Check gradle plugin.
```

Comment:
We throw an exception in the gradle plugin if the file doesn't exist, so feel free to fail if that's the case.

https://github.com/FirebasePrivate/firebase-android-buildtools/blob/0d775e83e2686f87378e62f8716d1cff71c353d4/firebase-appdistribution/appdistribution-gradle/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionOptions.java#L212

Reply:
yea that is what we thought, but we wanted to be sure. I have updated it so that if there is a path that is not correct an error will be thrown to behave similar to the gradle plugin.